Source/Src_nd/diag.F90

```diff
@@ -1,56 +1,58 @@
+! diag module for MAESTROeX. 
```

Comment:
Can you please return it to its previous location so it's compatible with `sphinx-fortran` (for the docs generation)

Reply:
All thats needed is module <module name> to be on line 1, then sphinx will pick up any comments between then and the next bit of code right? I think this should now be compatible